beets/importer.py

```diff
@@ -581,8 +581,13 @@ def handle_created(self, session):
     def lookup_candidates(self):
         """Retrieve and store candidates for this album.
         """
+        # Use a MusicBrainz id directly if provided by the importer -m option.
+        mb_ids = []
+        if config['import']['musicbrainz_ids'].exists():
+            mb_ids = config['import']['musicbrainz_ids'].get()
```

Comment:
The usual way to provide a default is to put it in `config_default.yaml` instead of using an explicit `.exists()` check.

Also, you might consider using a type check with `.get()` here, even if it seems somewhat redundant. `as_str_seq()` might be appropriate.